basicMysql/login/login.php

```diff
@@ -0,0 +1,69 @@
+<?php
+  session_start();
+?>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>User-Login</title>
+  <style>
+    <?php include "../../style.css" ?>
```

Comment:
We can also use a link tag for the CSS file instead of doing the same thing using PHP include.

Reply:
Yes but when I used `link` tag I facing isses while using the `CSS` on dynamic `HTML` part.